src/main/java/com/sss/garage/controller/driver/DriverController.java

```diff
@@ -24,17 +25,17 @@ public class DriverController extends SssBaseController {
     @GetMapping
     @Operation(operationId = "getAllDrivers", summary = "Get list of all drivers")
     @ResponseStatus(HttpStatus.OK)
-    public List<SimpleDriverDTO> getAllDrivers() {
-        return mapAsList(this.driverFacade.getAllDrivers(), SimpleDriverDTO.class);
+    public List<DetailedDriverDTO> getAllDrivers() {
```

Comment:
Tutaj raczej chcemy simple driver dto, jak na stronce bedziemy wyswietlac liste uzytkownikow to bez statystyk, elo itd (ale nie ma dizajnów więc troche lul)